tests/options_test.cc

```diff
@@ -59,47 +52,27 @@ TEST(OptionsLoadFileTest, ASimpleFile) {
 	EXPECT_TRUE(opt.loadOptionsFromFile("test_config"));
 }
 
-//MODULES TESTS
-TEST(OptionsLoadStreamModTest, WorkingSimpleCase) {
-	std::stringstream mod("server { module { type = echo; path = \"/testFiles1\"; }}");
+//HANDLERS TESTS
+/*TEST(OptionsLoadStreamHandTest, WorkingSimpleCase) {
+	std::stringstream mod("port 8080; path /echo EchoHandler {}");
+	std::stringstream mod2("port 8080; path / StaticHandler { root /foo/bar; }");
 
 	Options opt;
 	EXPECT_TRUE(opt.loadOptionsFromStream(&mod));
-}
-
-TEST(OptionsLoadStreamModTest, BadStatementSize) {
-	std::stringstream mod("server { module { = echo; } }");
-
-	Options opt;
-	EXPECT_FALSE(opt.loadOptionsFromStream(&mod));
-}
+	EXPECT_TRUE(opt.loadOptionsFromStream(&mod2));
+} */
 
-TEST(OptionsLoadStreamModTest, NoEquals) {
-	std::stringstream mod("server { module { type type echo; } }");
+TEST(OptionsLoadStreamHandTest, BadStatementSize) {
+	std::stringstream mod("port 8080; path /echo {}");
 
 	Options opt;
 	EXPECT_FALSE(opt.loadOptionsFromStream(&mod));
 }
 
-TEST(OptionsLoadStreamModTest, NotValidType) {
-	std::stringstream mod("server { module { type = shouldfail; } }");
+TEST(OptionsLoadStreamHandTest, NotValidType) {
+	std::stringstream mod("port 8080; module EchoHandler {}");
```

Comment:
This test passes for the wrong reason I think. Maybe change it to something like 
`path / NotARealHandlerType { }`